client/src/components/NavBar.jsx

```diff
@@ -26,6 +26,7 @@ const useStyles = makeStyles({
     backgroundColor: "white",
     textAlign: "center",
     position: "sticky",
+    height: "10vh",
```

Comment:
I give the navbar a fixed height to make calculations easier. It still needs to be made responsive for small screens (the sandwich button solution)